### PR TITLE
[FLINK-25783][docs-zh] Translate azure_table_storage.md page into Chinese.

### PR DESCRIPTION
**Pull request by RocMarshal**
## What is the purpose of the change

*A pure translation task.*

## Brief change log
Translate azure_table_storage.md page into Chinese.
file: flink/docs/content.zh/docs/connectors/datastream/formats/azure_table_storage.md

## Verifying this change
Translate azure_table_storage.md page into Chinese.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / **not documented**)

### CODE REVIEW

**Comment by RocMarshal** [comment]
@klion26 Hi, would you mind checking it if you have free time?  thx.

**Comment by gaoyunhaii** [comment]
Hi @RocMarshal sorry for missing the notification... I'll have a look very soon~

File: docs/content.zh/docs/connectors/datastream/formats/azure_table_storage.md [hunk 1]
```diff
@@ -27,24 +27,24 @@ under the License.
 
 # Azure Table Storage
 
-This example is using the `HadoopInputFormat` wrapper to use an existing Hadoop input format implementation for accessing [Azure's Table Storage](https://docs.microsoft.com/en-us/azure/storage/tables/table-storage-overview).
+本例使用 `HadoopInputFormat` 包装器使用现有的 Hadoop input format 实现访问 [Azure's Table Storage](https://docs.microsoft.com/en-us/azure/storage/tables/table-storage-overview).
```
Comment (gaoyunhaii) [inline docs/content.zh/docs/connectors/datastream/formats/azure_table_storage.md:30]: 包装器使用现有的 -> 包装器`来`使用现有的

File: docs/content.zh/docs/connectors/datastream/formats/azure_table_storage.md [hunk 2]
```diff
@@ -27,24 +27,24 @@ under the License.
...
 
-1. Download and compile the `azure-tables-hadoop` project. The input format developed by the project is not yet available in Maven Central, therefore, we have to build the project ourselves.
-   Execute the following commands:
+1. 下载并编译 `azure-tables-hadoop` 项目。该项目开发的 input format 在 Maven 中心尚不可用，因此，我们必须自己构建该项目。
```
Comment (gaoyunhaii) [inline docs/content.zh/docs/connectors/datastream/formats/azure_table_storage.md:32]: 在 Maven 中心尚不可用 -> 在 Maven 中心仓库尚不存在 ?

File: docs/content.zh/docs/connectors/datastream/formats/azure_table_storage.md [hunk 3]
```diff
@@ -59,13 +59,13 @@ curl https://flink.apache.org/q/quickstart.sh | bash
...
 ```
 
-`flink-hadoop-compatibility` is a Flink package that provides the Hadoop input format wrappers.
-`microsoft-hadoop-azure` is adding the project we've build before to our project.
+`flink-hadoop-compatibility` 是一个提供 Hadoop input format 包装器的 Flink 包。
+`microsoft-hadoop-azure` 可以将之前构建的部分添加到项目中。
 
-The project is now ready for starting to code. We recommend to import the project into an IDE, such as IntelliJ. You should import it as a Maven project.
-Browse to the file `Job.java`. This is an empty skeleton for a Flink job.
+项目现在已经可以开始编码。我们建议将项目导入 IDE，例如 IntelliJ。你应该将其作为 Maven 项目导入。
```
Comment (gaoyunhaii) [inline docs/content.zh/docs/connectors/datastream/formats/azure_table_storage.md:65]: 项目现在已经可以开始编码 -> 现在可以开始进行项目的编码 ?

File: docs/content.zh/docs/connectors/datastream/formats/azure_table_storage.md [hunk 4]
```diff
@@ -59,13 +59,13 @@ curl https://flink.apache.org/q/quickstart.sh | bash
...
+浏览到文件 `Job.java`。这是 Flink 作业的初始框架。
```
Comment (gaoyunhaii) [inline docs/content.zh/docs/connectors/datastream/formats/azure_table_storage.md:66]: 浏览到 -> 跳转到 ?

File: docs/content.zh/docs/connectors/datastream/formats/azure_table_storage.md [hunk 5]
```diff
@@ -114,15 +114,14 @@ public class AzureTableExample {
...
 
-    // execute program
+    // 执行程序
     env.execute("Azure Example");
   }
 }
 ```
-
-The example shows how to access an Azure table and turn data into Flink's `DataStream` (more specifically, the type of the set is `DataStream<Tuple2<Text, WritableEntity>>`). With the `DataStream`, you can apply all known transformations to the DataStream.
+该示例展示了如何访问 Azure 表和如何将数据转换为 Flink 的 `DataStream`（更具体地说，集合的类型是 `DataStream<Tuple2<Text, WritableEntity>>`）。使用 `DataStream`，你可以将所有已知的 transformations 应用到 DataStream。
```
Comment (gaoyunhaii) [inline docs/content.zh/docs/connectors/datastream/formats/azure_table_storage.md:125]: 使用 `DataStream`，你可以将所有已知的 transformations 应用到 DataStream。 
-> 
你可以将所有已知的 transformations 应用到这个 DataStream 实例。

**Review by gaoyunhaii — COMMENTED** [review]
Thanks @RocMarshal for the PR! I have left some comments.

**Comment by RocMarshal** [comment]
@gaoyunhaii Thanks a lot for the review. 
I make some change based on your comments. Looks better now.

**Comment by gaoyunhaii** [comment]
Thanks @RocMarshal for the update! LGTM The failed test should be irrelevant since this PR only affects docs.

**Comment by gaoyunhaii** [comment]
Merged on master via 3c65447651aa6c032c6752cf58b7bf719af52787

**Pull request closed without merging**